{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError\/internalFailure"},"metadata":{"modules":[{"name":"SotoCore"}],"symbolKind":"property","title":"internalFailure","externalID":"s:8SotoCore14AWSServerErrorV15internalFailureACvpZ","roleHeading":"Type Property","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"internalFailure","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore14AWSServerErrorV","text":"AWSServerError","kind":"typeIdentifier"}],"role":"symbol"},"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError"]]},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awsservererror\/internalfailure"]}],"schemaVersion":{"patch":0,"minor":3,"major":0},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"internalFailure","kind":"identifier"},{"kind":"text","text":": "},{"text":"AWSServerError","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError","preciseIdentifier":"s:8SotoCore14AWSServerErrorV","kind":"typeIdentifier"},{"kind":"text","text":" { "},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSServerError/internalFailure":{"kind":"symbol","abstract":[],"title":"internalFailure","url":"\/documentation\/sotocore\/awsservererror\/internalfailure","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError\/internalFailure","type":"topic","fragments":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"internalFailure"},{"kind":"text","text":": "},{"text":"AWSServerError","preciseIdentifier":"s:8SotoCore14AWSServerErrorV","kind":"typeIdentifier"}],"role":"symbol"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSServerError":{"kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AWSServerError"}],"abstract":[],"role":"symbol","title":"AWSServerError","url":"\/documentation\/sotocore\/awsservererror","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSServerError","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore":{"abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}],"title":"SotoCore","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","url":"\/documentation\/sotocore","kind":"symbol","type":"topic","role":"collection"}}}